librz/core/cmd_descs/cmd_descs.yaml

```diff
@@ -564,15 +564,15 @@ commands:
     options: "[?]"
     summary: Command specifiers (table-output only for now)
     type: RZ_CMD_DESC_TYPE_FAKE
-    usage: "<command>[:<table_spec>[:<table_spec>:...]:<output_spec>]"
+    usage: "<command>[:<table_spec>:<table_spec>...][:<output_spec>]"
```

Comment:
Doesn't this mark the optionality of the the second `<table_spec>`?

Reply:
Yes, one can be more precise with the brackets 3922da1882e8078579d44e854564c533dfdc9502